css/style.css

```diff
@@ -0,0 +1,213 @@
+body{
+    font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+    background: url(../img/your_verse_heroN.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height:475px;
+}
+
+#menu-outer {
+    margin-top: 47px;
+	height: 84px;
+	background: url(img/bar-bg.jpg) repeat-x;
+}
+
+.table {
+    display: table;   /* Allow the centering to work */
+    margin: 0 auto;
+    
+}
+
+
+ul#horizontal-list {
+    min-width: 696px;
+    width:940px; height:36px;
+    list-style: none;
+    border-radius:4px;
+    padding: 0;margin:0;
+    background:
+		-webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, .2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, .2))),
+		url('') no-repeat 50% 50%,
+		-webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080))
+	;
+	}
+ul#horizontal-list li {
+    display: inline;
+    padding: 0 26.6px;
+    padding-bottom: 8px;
+    margin: 0; 
+}
+ul#horizontal-list li:first-child img{
+    
+}
+ul#horizontal-list li:nth-child(n){
+    
+}
+ul#horizontal-list li{
+    display: inline-block;
+    float: left;
+    border-right: 1px solid black;
+    padding: 0 20px;
+    vertical-align: middle;
+    height: 30px;padding-top: 6px;
+}
+ul#horizontal-list li:first-of-type {
+    margin: 0;
+    padding: 0;
+    border-top-left-radius: 5px;
+    border-bottom-left-radius: 5px;
+ }
+ul#horizontal-list li:last-of-type {
+     padding-right: 0;
+     border-right: 0px solid black;
+ }
+ul#horizontal-list li:last-of-type:hover {
+     background-color: rgba(0, 0, 0, 0);
+ }
+ul#horizontal-list li a{
+     color:white;
+     text-decoration: none;
+ }
+ul#horizontal-list li:hover{
+    background-color: rgba(0, 0, 0, 0.466);
+}
+ul#horizontal-list li a > div{
+    background-image: url("../img/Apple-Logo.png");
+    background-size: 20px;
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 80px;
+    height: 35px;
+    margin-top: -22px;
+}
+ul#horizontal-list ul {
+    list-style-type: none;
+    overflow: hidden;
+    margin: 0 20px auto 0;
+    padding-left: 0;
+    padding-right: 0;
+    height: 100%;
+}
+ul#horizontal-list  a {
+    margin: 0 12px;
+}
+ul#horizontal-list input {
+    margin: -3px 10px;
+    background-color: rgba(255,255,255,.1);
+    background-image: url("../img/search_icon.png");
+    background-repeat: no-repeat;
+    background-position: 5px;
+    background-size: 14px;
+    border-radius: 12px;
+    box-shadow: rgba(0, 0, 0, 0.6) 0 2px 1px inset, rgba(255, 255, 255, 0.2) 0 1px 0;
+    border: 0px solid rgba(255,255,255,.1);
+    height: 22px;
+    padding: 0;
+    width:100px;
+}
+input {
+    -webkit-writing-mode: horizontal-tb !important;
```

Comment:
`SUGGESTION`

Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`. Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

Reply:
@bolah2009 for accepting my project :)
I removed !important

Reply:
Yay!